packages/core/src/components/TextField/Mask.example.jsx

```diff
@@ -0,0 +1,50 @@
+import TextField, { unmaskValue } from './TextField';
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { unmaskValue } from '@cmsgov/design-system-core';
+
+function handleBlur(evt, mask) {
+  console.log('Unmasked value:', unmaskValue(evt.target.value, mask));
+}
+
+const Example = () => {
+  return (
+    <div>
+      <TextField
+        ariaLabel="Enter monthly income amount in dollars."
+        label="Currency"
+        mask="currency"
+        name="currency_example"
+        onBlur={evt => handleBlur(evt, 'currency')}
+        value="2500"
+      />
+
+      <TextField
+        label="Phone number"
+        mask="phone"
+        name="phone_example"
+        onBlur={evt => handleBlur(evt, 'phone')}
+        type="tel"
+        value="1234567890"
+      />
+
+      <TextField
+        label="Social security number (SSN)"
+        mask="ssn"
+        name="ssn_example"
+        onBlur={evt => handleBlur(evt, 'ssn')}
+        value="123456789"
```

Comment:
Would it be out of scope to make these clip extra characters at the end? Like I shouldn't be able to input `123-45-678910111213`, and it would be nice if this feature handled that part as well as the additive formatting.

Reply:
It actually used to do that but I added support for unexpectedly long values. I think in these cases the consuming app should surface an error message rather than the masking functionality removing characters and potentially resulting in an incorrect value being submitted.